tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -295,7 +295,7 @@ EVAL=0
 
```

Comment:
Not sure if the useless cat is fixable.

    cat "$TMPFILE" | "$KDBCOMMAND" import $Mountpoint 2>/dev/null
        ^-- SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
                                          ^-- SC2086: Double quote to prevent globbing and word splitting.